### PR TITLE
COMMERCE-7597 Manage base price list exception

### PR DESCRIPTION
**Pull request by dsitu**
https://issues.liferay.com/browse/COMMERCE-7597

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/jaxrs/exception/mapper/BasePriceListCannotDeleteExceptionMapper.java [hunk 1]
```diff
@@ -39,13 +40,19 @@
...
-	public String getErrorDescription() {
-		return "Unable to delete base price list";
+	protected Problem getProblem(
+		CommerceBasePriceListCannotDeleteException
+			commerceBasePriceListCannotDeleteException) {
+
+		return new Problem(
+			null, Response.Status.BAD_REQUEST,
+			commerceBasePriceListCannotDeleteException.getMessage(),
+			"CommerceOrderValidatorException");
```
Comment (luca-pellizzon) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/jaxrs/exception/mapper/BasePriceListCannotDeleteExceptionMapper.java:50]: why is this one a "CommerceOrderValidatorException"?

**Review by luca-pellizzon — COMMENTED** [review]
Because of the changes to the dataset, we should ask for a review also to the frontend team.
Thanks

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/jaxrs/exception/mapper/BasePriceListCannotDeleteExceptionMapper.java (hunk 1 above)
Comment (dsitu) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/jaxrs/exception/mapper/BasePriceListCannotDeleteExceptionMapper.java:50]: oh nice catch, bad typo, I will fix that! thanks!

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
Closing to rework the solution. We want to avoid making changes to dataset in this bug ticket if we do not need to.

**Pull request closed without merging**